.github/workflows/create-release-workflow.yml

```diff
@@ -0,0 +1,60 @@
+on:
+  release:
+    types: [published]
+jobs:
+  add-assets-to-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - uses: actions/setup-go@v2
+      - name: Get git tag
+        id: tag_name
+        uses: little-core-labs/get-git-tag@v3.0.2
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.2.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+      - run: make gen_release
+      - name: Upload release (windows)
+        uses: actions/upload-release-asset@v1.0.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./build/factorio-server-manager-windows.zip
+          asset_name: factorio-server-manager-windows-${{ steps.tag_name.outputs.tag }}.zip
+          asset_content_type: application/octet-stream
+      - name: Upload release (linux)
+        uses: actions/upload-release-asset@v1.0.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./build/factorio-server-manager-linux.zip
+          asset_name: factorio-server-manager-linux-${{ steps.tag_name.outputs.tag }}.zip
+          asset_content_type: application/octet-stream
+  docker-push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - uses: actions/setup-go@v2
+      - uses: docker/setup-buildx-action@v1
+      - name: Get git tag
+        id: tag_name
+        uses: little-core-labs/get-git-tag@v3.0.2
+      - name: Login to dockerhub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
We need a service account for dockerhub uploads.

Reply:
Currently i use my account (on my fork) and would also add it to this repo, i dont see the need of creating an additional dockerhub account.

Reply:
Better to make a service account with limited access then to use your personal account for the uploads.